lang/cs/Org.Apache.REEF.Driver/Evaluator/EvaluatorRequest.cs

```diff
@@ -29,7 +29,7 @@ namespace Org.Apache.REEF.Driver.Evaluator
     internal class EvaluatorRequest : IEvaluatorRequest
     {
         internal EvaluatorRequest()
-            : this(0, 0, 1, string.Empty, Guid.NewGuid().ToString("N"), string.Empty, Enumerable.Empty<string>().ToList(), true, String.Empty)
```

Comment:
Do the changes in this file actually change something material related to the goals of this PR? If not, please remove them and move them to a separate PR.

Reply:
Correct. Unrelated to this PR. Will send a different pull request if needed.